types/lhr.d.ts

```diff
@@ -48,6 +51,15 @@ interface Result {
 
 // Result namespace
 declare module Result {
+  /**
+   * The full output of a Lighthouse flow. Includes a series of Lighthouse runs.
+   * TODO(FR-COMPAT): Add flow specific metadata (e.g. interaction steps).
+   */
+  interface FlowResult {
```

Comment:
I've been thinking about using `fr` as the *official* shorthand for this, so the flow version of a "Lighthouse result" (`lhr`) would be a "flow result" (`fr`). This is also the shorthand for "Fraggle Rock", and I'm not sure if that will be helpful or confusing.

Reply:
`lhfr`? it's still lighthouse ;)

Reply:
I'm not really a fan of using `lhfr.lhrs` since `lhfr` and `lhrs` look too similar, but I could probably get used to it.

Reply:
`lhfr.lhrs` is indeed rough. Maybe we just actually call these `flowResult`?

`result` just felt too vague on its own which is how it became **L**ight**h**ouse **r**esult, but with the modifier it's not that bad.

Reply:
I  also don't know if you want it as `LH.Result.FlowResult` :)

Reply:
> I also don't know if you want it as LH.Result.FlowResult

Changed this to `LH.FlowResult`